pkgs/by-name/vs/vscode-js-debug/package.nix

```diff
@@ -0,0 +1,88 @@
+{ lib
+, buildNpmPackage
+, fetchFromGitHub
+, jq
+, libsecret
+, pkg-config
+, nodePackages
+, runCommand
+, vscode-js-debug
+, nix-update-script
+}:
+
+buildNpmPackage rec {
+  pname = "vscode-js-debug";
+  version = "1.88.0";
+
+  src = fetchFromGitHub {
+    owner = "microsoft";
+    repo = "vscode-js-debug";
+    rev = "v${version}";
+    hash = "sha256-gmeuRUcdz/4+FtPOblNj5DX3otXNRjHJjhPcCRuWXAY=";
+  };
+
+  npmDepsHash = "sha256-M4h2p8GLVjBDla0ile1jKWF6wPSdgcumx2GKm9KGmlw=";
+
+  nativeBuildInputs = [ pkg-config nodePackages.node-gyp ];
+
+  buildInputs = [ libsecret ];
+
+  postPatch = ''
+    ${jq}/bin/jq '
```

Comment:
```suggestion
    ${lib.getExe jq} '
```

Reply:
Adding `jq` to `nativeBuildInputs` works now, so have done that instead.